scss/main.scss

```diff
@@ -290,23 +290,60 @@ h1,h2,h3 {
   margin-bottom: 2rem;
   padding-left: 2rem;
   padding-right: 2rem;
+  p.text-center {
+    margin-bottom: 48px;
+  }
 }
 
 #meetupLink {
   color: $cfa-blue;
 }
 
+.meetup-content {
+  margin-bottom: 1.5rem;
+  > * {
+  margin-bottom: 1rem;
+  }
+}
+
+
+
 .rsvp-stuff {
   margin-top: 1rem;
+  p {
+    font-family: $font-family-alt;
+    font-weight: 500;
+    color: $darker-grey;
+    font-size: .875rem; 
+  }
 }
 
 .attendees {
   border-bottom: 2rem solid transparent;
 }
 
 .attend-button {
+  font-family: $font-family-alt;
+  font-weight: 600;
+  color: white; 
+  font-size: .875rem;
   border-bottom: 1rem solid transparent;
   padding-left: 5%;
+  @media (max-width: 768px) {
+    padding: 0 .9375rem; 
+    a {
+      display: inline-block; 
+      width: 100%; 
+    }
+  }
+  
+  a:link, a:hover {
+    color: $white;
```

Comment:
I think the anchor tag will always be white, correct?
We're currently not changing the :visited style, so the text disappears after you visit the link:

<img width="638" alt="Screen Shot 2021-02-18 at 8 33 49 AM" src="https://user-images.githubusercontent.com/9326234/108372796-c9512880-71c4-11eb-9d43-cc70ae02162b.png">

Reply:
I am finding some strange behavior. It seems that something is overriding the color property of anchor tags throughout the file and turning them red, which has forced a lot of repetitive color rules like the one you quote above. When I remove the code you quote above or try to otherwise root out the repetition in the file, text starts turning red all over the place. I hate to leave the styles so repetitive, but the simplest solution I've found so far is just adding a:visited to the above rule, preventing the text from turning red. Any thoughts on what could be causing this?

Reply:
I think what it should be is:
```suggestion
    a, a:link, a:hover {
      color: $white;
    }
```
Actually I don't know what a:link specifically is doing ... so maybe its redundant? But setting generic a tag seems to fix the behavior you're seeing on my end.

Reply:
@albachteng You'll find the global anchor style in the `_reboot.scss` file. The color is currently set to `$cfa-red`. So, we'll need to override that, since this anchor seems to be a special case.

The `:link` pseudo-class targets any tag with an `href` attribute, that hasn't been visited.
Can we change the selector to just `a`, and ditch all of the pseudo-class selectors?

Reply:
Thank you both @jrapala @daleinen7, I see what was happening now. I've removed the redundant rules and think we should be good to go